main.css

```diff
@@ -386,17 +365,268 @@ form > textarea:focus {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: url(./assets/img/TwoCircles.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .headline--title {
+    width: 70vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .headline--container h1 {
+    width: auto;
+    left: 0;
+    top: 0;
+    color: var(--primary-1);
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 9rem;
+  }
+
+  .headline--container h2 {
+    font-size: 4rem;
+    width: auto;
+    padding-top: 3rem;
+    padding-bottom: 6rem;
+  }
+
+  .headline--container .description {
+    width: 55%;
+    text-align: center;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 2rem;
+    line-height: 2.8rem;
+    margin-top: 4rem;
+    padding-bottom: 7rem;
+  }
+
+  .skills--container {
+    width: 35vw;
+    height: auto;
+    padding: 1rem;
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+  }
+
+  .skills--item {
+    width: 10rem;
+    margin-right: 3rem;
+    margin-bottom: 3rem;
+  }
+
+  .supporting-item-container {
+    width: 80vw;
+    margin-top: 12rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .social--icons--container {
+    min-width: 200px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .supporting-item-l {
+    display: inline;
+  }
+
+  .supporting-item {
+    display: inline;
+    width: 77px;
+    height: 40px;
+    left: 61.5px;
+    top: 64px;
+    margin: 0;
+  }
+
+  .recent--work--container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 20rem;
+    background-image: url(./assets/svg/Group\ 65.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .mobile--title {
+    display: none;
+  }
+
+  .section--tittle .desktop--title {
+    display: inline-block;
+    width: 80vw;
+    align-self: baseline;
+    padding-bottom: 3rem;
+    font-family: var(--ff);
+    text-align: left;
+    color: var(--primary-1);
+    font-size: 6.4rem;
+    font-weight: 700;
+    border-bottom: solid 1px white;
+  }
+
+  .cards--container {
+    width: 80vw;
+    margin-top: 2rem;
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto auto auto;
+  }
+
+  .recent--work--card {
+    grid-column: 1/4;
+    width: inherit;
+    padding-top: 2.5rem;
+    padding-bottom: 8rem;
+    display: flex;
+  }
+
+  .card--thumbnail {
+    flex: 3;
+  }
+
+  .card--thumbnail:nth-child(odd) {
+    margin-right: 3rem;
+  }
+
+  .card--thumbnail:nth-child(even) {
+    margin-left: 3rem;
+  }
+
+  .card--thumbnail img {
+    width: 100%;
+    height: auto;
+  }
```

Comment:
[Optional]

- [ ] If you check the resolutions above 1400px, you can see that images become too big and overflow their containers. It will be better if you set max-width for the container to prevent this

![Screenshot 2021-06-11 at 01-36-34 Alef's Portfolio](https://user-images.githubusercontent.com/1642014/121607443-b2c32b80-ca58-11eb-89e0-3753682278ab.png)